### PR TITLE
Add and setup Doxygen configuration file

### PR DESCRIPTION
**Pull request by szledan**
Signed-off-by: Szilard Ledan <szledan@gmail.com>

### CODE REVIEW

**Comment by kkristof** [comment]
lgtm

File: tools/scripts/update-docs.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# The views and conclusions contained in the software and documentation are those
+# of the authors and should not be interpreted as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
+
+cd docs
```
Comment (dbatyai) [inline tools/scripts/update-docs.sh]: It would be better to operate relative to the base directory, instead of the docs directory.

**Comment by szledan** [comment]
I have been renamed the configuration file from `geprad-release.doxygen` to `gepard-public.doxygen` and use relative operate in updater script.
@kkristof & @dbatyai , could you check it, please?

**Comment by kkristof** [comment]
It still looks good to me :) +1

**Comment by dbatyai** [comment]
lgtm

**Pull request merged**